README.md

```diff
@@ -25,35 +25,38 @@ For more details on how to use this package, check out example documentation bel
-| [contributte-event-dispatcher](https://github.com/contributte/playground/tree/master/contributte-event-dispatcher) | Example of dispatching and subscribing events. |
-| [contributte-forms-bootstrap](https://github.com/contributte/playground/tree/master/contributte-forms-bootstrap) | Example of contributte/forms-bootstrap usage. |
-| [contributte-live-form-validation](https://github.com/contributte/playground/tree/master/contributte-live-form-validation) | Example of live form validation. |
+| apitte-fullstack - moved to https://github.com/contributte/apitte-skeleton | Fullstack application with example of all plugins. |
+| apitte-micro - moved to https://github.com/contributte/apitte-skeleton | Micro application with direct dispatching. |
+| apitte-simple - moved to https://github.com/contributte/apitte-skeleton | Simple middleware application. |
+| apitte-nette-simple - moved to https://github.com/contributte/apitte-skeleton | Simple Nette |
+| [contributte-anabelle](https://github.com/contributte/playground/tree/master/contributte-anabelle) | Example of contributte-anabelle usage. |
+| contributte-api-router - moved to https://github.com/contributte/api-router-skeleton | Starter project for ApiRouter. |
+| contributte-console - moved to https://github.com/contributte/console-skeleton | Example of writing own commands and how to install console-extra with all commands. |
+| contributte-datagrid - moved to https://github.com/contributte/datagrid-skeleton | Dockerized example of our datagrid. [DEMO](https://examples.contributte.org/packages/datagrid/) |
+| contributte-event-dispatcher moved to https://github.com/contributte/webapp-skeleton | Example of dispatching and subscribing events. |
+| contributte-forms-bootstrap - moved to https://github.com/contributte/gui-skeleton | Example of contributte/forms-bootstrap usage. |
+| contributte-live-form-validation - moved to https://github.com/contributte/gui-skeleton | Example of live form validation. |
 | [contributte-mailing](https://github.com/contributte/playground/tree/master/contributte-mailing) | Example of contributte/mailing usage. |
 | [contributte-neonizer](https://github.com/contributte/playground/tree/master/contributte-neonizer) | Example of contributte/neonizer usage. |
 | [contributte-pdf ](https://github.com/contributte/playground/tree/master/contributte-pdf) | Example of contributte/pdf usage. |
-| [contributte-reCAPTCHA](https://github.com/contributte/playground/tree/master/contributte-reCAPTCHA) | Example of Google reCaptcha. |
+| contributte-reCAPTCHA - moved to https://github.com/contributte/gui-skeleton | Example of Google reCaptcha. |
 | [contributte-thepay](https://github.com/contributte/playground/tree/master/contributte-thepay) | Example of contributte/thepay usage. |
 | [nextras-orm-generator](https://github.com/contributte/playground/tree/master/nextras-orm-generator) | Example of nextras entity generation |
```

Comment:
Ukazku orm generatoru prehodit do contributte/nextras-orm-generator do slozky example

Reply:
Kam že to? Nerozumím.

Reply:
V repozitari contributte/nextras-orm-generator vytvorit slozku example a tam to presunout.

Reply:
ok, hotovo